nodes/nodes/ecosystems/eco-self-node-remediation-operator.adoc

```diff
@@ -13,25 +13,25 @@ include::modules/eco-self-node-remediation-operator-about.adoc[leveloffset=+1]
 include::modules/eco-self-node-remediation-about-watchdog.adoc[leveloffset=+2]
 
 .Additional resources
-xref:../../virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc#virt-configuring-a-watchdog[Configuring a watchdog]
+xref:../../../virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc#virt-configuring-a-watchdog[Configuring a watchdog]
 
 include::modules/eco-self-node-remediation-operator-control-plane-fencing.adoc[leveloffset=+1]
 
 include::modules/eco-self-node-remediation-operator-installation-web-console.adoc[leveloffset=+1]
 
 include::modules/eco-self-node-remediation-operator-installation-cli.adoc[leveloffset=+1]
 
-include::modules/eco-configuring-machine-health-check-with-self-node-remediation.adoc[leveloffset=+1]
+include::modules/eco-self-node-remediation-operator-configuring.adoc[leveloffset=+1]
 
 include::modules/eco-self-node-remediation-operator-troubleshooting.adoc[leveloffset=+1]
 
 [id="gather-data-self-node-remediation"]
 == Gathering data about the Self Node Remediation Operator
-To collect debugging information about the Self Node Remediation Operator, use the `must-gather` tool. For information about the `must-gather` image for the Self Node Remediation Operator, see xref:../../support/gathering-cluster-data.adoc#gathering-data-specific-features_gathering-cluster-data[Gathering data about specific features].
+To collect debugging information about the Self Node Remediation Operator, use the `must-gather` tool. For information about the `must-gather` image for the Self Node Remediation Operator, see xref:../../../support/gathering-cluster-data.adoc#gathering-data-specific-features_gathering-cluster-data[Gathering data about specific features].
 
 [role="_additional-resources"]
 [id="additional-resources-self-node-remediation-operator-installation"]
 == Additional resources
 
-* The Self Node Remediation Operator is supported in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
-* xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster]
+* The Self Node Remediation Operator is supported in a restricted network environment. For more information, see xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
```

Comment:
Same comment as previous. No text in these sections.

Reply:
Thanks @mjpytlak I've applied your feedback.